ttab/scenarios/default_scenarios.py

```diff
@@ -36,4 +36,232 @@
             intra_domain_shuffle=True,
         ),
     ),
+    "PTTA_cifar10": Scenario(
```

Comment:
Too much unnecessary repetition. We'd better develop some new code to support this particular setting.